docker/db/scripts/20220725_test_migration.sql

```diff
@@ -0,0 +1,16 @@
+ INSERT INTO "Address"("id", "createdAt", "updatedAt","firstLine","secondLine","city","postCode","countryId") VALUES
```

Comment:
FYI -- this needs to be applied manually via the client or by attaching the shell (cos the migrations need to be run first). And probably best not to put this in the Prisma directory because this is just for test data that can be easily deleted without making Prisma panic